framework/db/mysql/QueryBuilder.php

```diff
@@ -390,7 +390,15 @@ private function getColumnDefinition($table, $column)
      */
     private function supportsFractionalSeconds()
     {
-        $version = $this->db->getSlavePdo()->getAttribute(\PDO::ATTR_SERVER_VERSION);
+        // use cache to prevent open mysql connection
+        // https://github.com/yiisoft/yii2/issues/13749#issuecomment-481657224
+        $key = [__METHOD__, $this->db->dsn];
```

Comment:
Shouldn't we also include `$this->db->username` here, same as in `Command::getCacheKey()`?

Reply:
i think mysql server version is only related with dsn